app/jobs/import_content_item_job.rb

```diff
@@ -12,9 +12,32 @@ def perform(*args)
     content_id = args[0]
 
     attributes = content_items_service.fetch(content_id)
-    attributes = attributes.merge(content_items_service.links(content_id))
+    links = content_items_service.links(content_id)
```

Comment:
For testing this, I've been calling `ImportContentItemJob.perform` from the console, and I think that has been working fine.